proto/p4/p4runtime.proto

```diff
@@ -168,6 +168,18 @@ message TableEntry {
   // default action of the table. If true, the "match" field must be empty and
   // the "action" field must be populated with a valid direct action.
   bool is_default_action = 8;
+  // The TTL for the entry, in nanoseconds. A value of 0 means that the table
+  // entry never "expires".
+  int64 idle_timeout_ns = 9;
+  message IdleTimeout {
+    // Time elapsed - in nanoseconds - since the table entry was last "hit" as
+    // part of a dataplane table lookup.
+    int64 elapsed_ns = 1;
+  }
+  // Table wite: this field should be left unset.
+  // Table read: if the table supports idle timeout and time_since_last_hit is
+  // set in the request, this field will be set in the response.
```

Comment:
What does it mean in a streaming notification?

Reply:
I'll update the comment for `IdleTimeoutNotifcation` to indicate that only the match key is required for members of the `table_entry` repeated field. I don't think there is value in including other fields (we probably just want to identify which entry has expired so we can remove it), including the `time_since_last_hit` field.